edx_proctoring/tests/test_views.py

```diff
@@ -1118,6 +1118,7 @@ def test_multiple_status_filters(self):
         (ProctoredExamStudentAttemptStatus.ready_to_submit,
             InstructorDashboardOnboardingAttemptStatus.proctoring_started),
         (ProctoredExamStudentAttemptStatus.submitted, InstructorDashboardOnboardingAttemptStatus.submitted),
+        (ProctoredExamStudentAttemptStatus.rejected, InstructorDashboardOnboardingAttemptStatus.rejected),
```

Comment:
Looks like you have a failed test. Please fix.

Reply:
This test isn't failing locally for some reason. Going to see if I can figure out why.

Reply:
I think `MockGradesService` and `MockCertificateService` were getting torn down inbetween test suites, so added those fixed it.